packages/backend/convex/http.ts

```
@@ -0,0 +1,80 @@
+router.route({
+  path: '/clerk-webhook',
+  method: 'POST',
+  handler: httpAction(async (ctx, req) => {
+    const event = await validateWebhook(req);
+    if (!event) {
+      return new Response('Invalid webhook', { status: 400 });
+    }
+
+    switch (event.type) {
+      case 'subscription.updated':
+        const subscription = event.data as SubscriptionUpdatedEventData;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)